recipes/blast/build.sh

```diff
@@ -55,6 +49,7 @@ export AR="${AR} rcs"
 ./configure \
     --with-dll \
     --with-mt \
+    --with-openmp \
```

Comment:
Nice. Does that make an impact to performance?

Reply:
In some cases, yes.